src/lib/components/inbox/bids/index.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+import styled from "styled-components/native"
+
+import { View } from "react-native"
+import { LargeHeadline } from "../typography"
+import ActiveBid from "./active_bid"
+
+const Container = styled.View`
+  margin: 20px 0 40px;
+`
+
+class ActiveBids extends React.Component<any, any> {
+  renderRows() {
+    const me = this.props.me || { lot_standings: [] }
+    const bids = me.lot_standings.map(bidData => {
+      return <ActiveBid key={bidData.active_bid.__id} bid={bidData} />
+    })
```

Comment:
If you felt like golfing, this could be: 
```js
const bids = me.lot_standings.map(bidData => <ActiveBid key={bidData.active_bid.__id} bid={bidData} />)
```